### PR TITLE
add sort, filter, pagination and highlight selection to log table

### LINKED ISSUE #16: Selected row in the table should be highlighted
When a row in the log table is clicked the log data is shown in the json viewer, and a location indicated on the map.  There is on indication though _which_ log line has been clicked.

It should be relatively straightforward to add this to the table view.

### PR DESCRIPTION
**Pull request by crystalzha0**
Fixes #16 #24 

- Selected table will be highlighted
- Clicking on table header cell will sort the corresponding column
- Each column can be filtered by input string

Extra:
- Added pagination so UI doesn't slow down when number of entries is very large

### CODE REVIEW

**Review by ewbao — APPROVED** [review]
LGTM pending James' review.

File: src/LogTable.js [hunk 1]
```diff
@@ -35,42 +35,187 @@ const Styles = styled.div`
       }
     }
   }
+
+  .logtable-head {
+    display: flex;
+  }
+  .logtable-row:hover {
+    background-color: #e6e6e6;
```
Comment (jamesjhan) [inline src/LogTable.js:43]: just making sure - these are from designs?

**Review by jamesjhan — APPROVED** [review]
not a huge comment

File: src/LogTable.js (hunk 1 above)
Comment (crystalzha0) [reply src/LogTable.js:43]: No, I added this for a better user experience.

**Pull request merged**